src/content/create-store.js

```diff
@@ -32,9 +32,7 @@ export default function initializeStore(): Store {
   }
 
   const store = createStore(
-    combineReducers(Object.assign({}, reducers, {
-      worker,
-    })),
+    combineReducers(reducers),
```

Comment:
nit: if you remove `worker` from here (TBH I didn't understand why it was for :) ), then you can call `combineReducers` directly in `reducers/index` for consistency with all reducers, so that `reducers` is already a reducer and not an object of reducers.